### DISCUSSION TITLE
Quick question about clearing requests

### DISCUSSION THREAD

**Discussion by blubottom** [seerr-team/seerr#678] [General]
Hey all,
Im running a full jellyfin/seerr/sonarr/radarr setup on ubuntu server / docker
Quick question, when someone requests something, it appears on the requests list. Is there a way to delete the request once it becomes available? I know I can filter by available, unavailable, failed etc but just seeing if there is a way to clear the request once it is available automatically to clean up the requests.

**Comment by bl4zee1g** [comment]
you found any solution to this?

**Comment by fallenbagel** [comment]
Jellyseerr is not really designed to remove requests after it becomes available as that would break other modules that are tied to the request history. So you can't make it auto delete available requests

**Reply by gmcoates101** [reply]
Could you clarify which other modules and what the impact would be on them please?

**Reply by realestninja** [reply]
would also like to understand this dependency. To me it would make sense to delete requests once the items become available. why is there are request for something that is available?
